src/components/InfoMenu.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div>
+    <div id="app-info">
+      <div class="common fill1" />
+      <div class="common info">
+        <InfoButton :buttonText="$t('$serviceInfoTitle')" @show-info="onShowInfo" />
+        <InfoButton :buttonText="$t('$instructionsTitle')" @show-info="onShowInstructions" />
+        <InfoButton :buttonText="$t('$termsOfServiceTitle')" @show-info="onShowTerms" />
+      </div>
+      <div class="common fill2" />
+      <div id="info-text">
+        <div v-if="showInfo">
+          <h2>{{$t('$serviceInfoTitle')}}</h2>
+          <p>{{$t('$serviceInfo')}}</p>
+        </div>
+        <div v-if="showInstructions">
+          <h2>{{$t('$instructionsTitle')}}</h2>
+          <p>{{$t('$instructions')}}</p>
+        </div>
+        <div v-if="showTerms">
+          <h2>{{$t('$termsOfServiceTitle')}}</h2>
+          <p>{{$t('$termsOfService')}}</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import InfoButton from '@/components/InfoButton.vue';
+
+@Component({
+  components: {
+    InfoButton,
+  },
+})
+export default class InfoMenu extends Vue {
+  public showInfo: boolean = false;
+  public showInstructions: boolean = false;
+  public showTerms: boolean = false;
+
+  public onShowInfo(isPressed: boolean) {
+    this.showInfo = isPressed;
+  }
+
+  public onShowInstructions(isPressed: boolean) {
+    this.showInstructions = isPressed;
+  }
+
+  public onShowTerms(isPressed: boolean) {
+    this.showTerms = isPressed;
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import "@/assets/styles/variables.scss";
+#app-info {
+  display: grid;
+  grid-template:
+    "fill1 div fill2" 4.4rem
```

Comment:
Eiks tää ois sama tehdä ". div ." ?

Reply:
Tarkotatko "div div div"? Sitten olis kyllä hankalampi asetella noi napit samaan linjaan sisällön kanssa.

Reply:
Aa joo katoin että nää ois rakennettu hieman eri tavalla. Katoin että nää napit oli omassa divissään ja tekstisisältö omassaan, mutta eihän se niin mennytkään. Jos tekis silleen nii sit ei tarviis näitä fill-divejä lainkaan ja tyylittely ois ehkä selkeempi kun ei tarviis tommosta .common-luokkaa.

Reply:
![image](https://user-images.githubusercontent.com/17126867/74160417-0c79d280-4c26-11ea-852a-e1eeda92df1b.png)
Aattelin siis näin et 1 sisältäis ton sinisen taustan ja 2 ois pelkästään tekstin näyttämiselle

Reply:
...Eiku niinhän se just meneekin :D Mitähän mä selitän. Anyway, vois muuttaa niin että `#app-info` sisältäis taustavärin ym. niin sit voi siivota noi fill-divit poikkee

Reply:
En mä ymmärrä...

Reply:
Teen ehdotelman

Reply:
Huomenna.. :p

Reply:
Elikkäs noin aattelin jotenkin. Oisko simppelimpi?

Reply:
Näppärää :)